multicluster/apis/multicluster/v1alpha1/multiclusterconfig_types.go

```diff
@@ -31,14 +31,16 @@ const (
 
```

Comment:
@luolanzone : I think we should use "internal" and "external" to be consistent with Node IP types. Could we add "internal" / "external" and meanwhile keep supporting "private" / "public“？

Reply:
sure, I will refine it to support both.

Reply:
I just checked the code, I feel it would be better to include the change in this PR. I believe you just need to add two more const here: https://github.com/antrea-io/antrea/blob/main/multicluster/apis/multicluster/v1alpha1/multiclusterconfig_types.go#L27-L30.

Reply:
We need to change the backend handling (decide which IP to use based on the option) right.

Reply:
oh, yes, you are right, you can leave it. I will update it later.